jupyter-notebooks/analysis-ready-data/ard_1_intro_and_best_practices.ipynb

```diff
@@ -202,9 +196,48 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'__daily_email_enabled': False,\n",
+       " '_links': {'_self': 'https://api.planet.com/data/v1/searches/1d3871654d6341ccad0e0f5d9274da11',\n",
+       "  'results': 'https://api.planet.com/data/v1/searches/1d3871654d6341ccad0e0f5d9274da11/results'},\n",
+       " 'created': '2023-06-22T11:01:08.808024Z',\n",
+       " 'filter': {'config': [{'config': {'coordinates': [[[-93.299129, 42.699599],\n",
+       "       [-93.299674, 42.812757],\n",
+       "       [-93.288436, 42.861921],\n",
+       "       [-93.265332, 42.924817],\n",
+       "       [-92.993873, 42.925124],\n",
+       "       [-92.993888, 42.773637],\n",
+       "       [-92.998396, 42.754529],\n",
+       "       [-93.019154, 42.699988],\n",
+       "       [-93.299129, 42.699599]]],\n",
+       "     'type': 'Polygon'},\n",
+       "    'field_name': 'geometry',\n",
+       "    'type': 'GeometryFilter'},\n",
+       "   {'config': {'gte': 90.0},\n",
+       "    'field_name': 'clear_percent',\n",
+       "    'type': 'RangeFilter'},\n",
+       "   {'config': {'gt': '2020-01-01T00:00:00Z', 'lt': '2020-01-31T00:00:00Z'},\n",
+       "    'field_name': 'acquired',\n",
+       "    'type': 'DateRangeFilter'}],\n",
+       "  'type': 'AndFilter'},\n",
+       " 'id': '1d3871654d6341ccad0e0f5d9274da11',\n",
+       " 'item_types': ['PSScene'],\n",
+       " 'last_executed': None,\n",
+       " 'name': 'temp_search',\n",
+       " 'search_type': 'saved',\n",
+       " 'updated': '2023-06-22T11:01:08.808024Z'}"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "# Let's look at our search request.\n",
     "# Note: This is just the request's structure, the search hasn't been implemented yet\n",
```

Comment:
We either type out the variable or explicitly call `print` or `pprint` to print it out. Let's be consistent. I'd recommend calling `pprint` for `request`.

Reply:
This also applied for everywhere else we print in this notebook. `request` isn't the only spot where we do this.